fonts.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+	font-family: "GothamPro";
+	    src: 
+		   url("./fonts/GothamPro-Bold.woff2") format("woff2"),
+		   url("./fonts/GothamPro-Bold.woff") format("woff");
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5896505)